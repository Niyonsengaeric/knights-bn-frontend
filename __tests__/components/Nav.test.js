import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import renderer from 'react-test-renderer';
import NavAuth from '../../src/components/NavAuth';

describe('Test Nav component', () => {
  it('should render Nav component', () => {
    const NavComponent = renderer.create(
      <Router>
        <NavAuth />
      </Router>,
    ).toJSON();
    expect(NavComponent).toMatchSnapshot();
  });
});
