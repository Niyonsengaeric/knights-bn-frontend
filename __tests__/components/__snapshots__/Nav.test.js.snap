// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Nav component should render Nav component 1`] = `
<div
  className="nav "
>
  <select
    className="lang lang--list"
    onChange={[Function]}
  >
    <option>
      Choose language.1
    </option>
    <option
      value="ki"
    >
      ikinyarwanda 
    </option>
    <option
      value="en"
    >
      english
    </option>
    <option
      value="fr"
    >
      fran√ßais
    </option>
  </select>
  <a
    href="/"
    onClick={[Function]}
  >
    <h3
      className="text-center pt-2 li"
    >
      <img
        className="logo_Auth"
        src=""
      />
    </h3>
  </a>
  <ul
    className="nav-link"
  >
    <a
      href="/login"
      onClick={[Function]}
    >
      <li
        className="li"
      >
        Login.1
      </li>
    </a>
    <a
      href="/signup"
      onClick={[Function]}
    >
      <li
        className="li"
      >
        Sign up.1
      </li>
    </a>
  </ul>
</div>
`;
